packages/chat-client/src/client.ts

```diff
@@ -64,13 +64,6 @@ export class ChatClient extends IChatClient {
       CHAT_CLIENT_STORAGE_PREFIX,
       (invite: ChatClientTypes.ReceivedInvite) => invite.id
     );
-    this.chatSentInvites = new Store(
```

Comment:
Context? Seems like you’re also nuking another store entirely? 😅

We should probs clean up the `CHAT_SENT_INVITES_CONTEXT` too if no longer needed.

Reply:
it's defined twice, I just removed the redundant one